R/rcodesign/build_tarballs.jl

```diff
@@ -24,35 +23,36 @@ if [[ "${target}" == x86_64-apple-darwin* ]]; then
     cp -ra System "/opt/${target}/${target}/sys-root/."
     popd
 fi
-cd $WORKSPACE/srcdir/PyOxidizer-apple-codesign-*/apple-codesign/
-cargo build --release
-install -Dvm 755 "../target/${rust_target}/release/rcodesign${exeext}" "${bindir}/rcodesign${exeext}"
-install_license ../LICENSE
+
+cd $WORKSPACE/srcdir/apple-platform-rs-apple-codesign-$version/
+cargo build --release --package apple-codesign --target-dir ${WORKSPACE}/tmp
+
+install_license $WORKSPACE/srcdir/apple-platform-rs-apple-codesign-$version/apple-codesign/LICENSE
```

Comment:
```suggestion
install_license apple-codesign/LICENSE
```

Reply:
Again this is here because of the use of the source

Reply:
then
```suggestion
install_license apple-codesign/LICENSE
```
looks a lot simpler to me